app/views/member/details/edit.html.haml

```diff
@@ -0,0 +1,37 @@
+%section#banner
```

Comment:
@despo can all the strings in this view be moved into translation files?

Reply:
Done